Misc/NEWS.d/next/Core_and_Builtins/2024-12-04-22-14-40.gh-issue-127119._hpyFE.rst

```diff
@@ -0,0 +1 @@
+Slightly optimize the :class:`int` deallocator.
```

Comment:
Maybe include the benchmarks results (it's a 4% improvement which is still noticable IMO). You should mention that it only concerns PGO builds as well.